test/unit/item_test.rb

```diff
@@ -82,7 +82,7 @@ class ItemTest < ActiveSupport::TestCase
   test 'should not update amount one percent if category type is not one percent' do
     #given
     item = items(:one)
-    amount = 2
+    amount = 30
```

Comment:
```suggestion
    amount = 37
```

Bez poprawki wywala taki błąd:
```
Error:
ItemTest#test_should_not_update_amount_one_percent_if_category_type_is_not_one_percent:
ActiveRecord::RecordInvalid: Negatywne sprawdzenie poprawności: Items Suma z dotacji (37) musi mieścić się w wartości wpisu (30.0)
    test/unit/item_test.rb:91:in `block in <class:ItemTest>'
```